examples/rust_control_interface/Dockerfile

```diff
@@ -14,10 +14,11 @@ RUN apk update && apk add --update-cache \
 COPY api /workspaces/build/api
 COPY examples/rust_control_interface /workspaces/build
 WORKDIR /workspaces/build
-RUN cargo build --release
+RUN --mount=type=cache,target=./target/release cargo build --release \
```

Comment:
Fixed it by providing absolute path. It is because of podman 4.3 inside the container does not support relative path. Outside of the container, it seems like I have newer podman version (4.6) which supports it. Thank you, fixed with the next commit.